akka-javasdk/src/main/java/akka/javasdk/agent/Agent.java

```diff
@@ -65,6 +65,8 @@ interface Builder {
 
       Builder systemMessage(String message);
 
+      Builder systemMessageFromTemplate(String templateId);
```

Comment:
we could add varargs for parameter replacement (using String.formatted)

Reply:
ok, I will add that in the following PRs